netbench/README.md

```diff
@@ -20,8 +20,24 @@ Here are a few examples of questions that netbench aims to answer:
 * How does certificate chain length affect handshake throughput?
 * Is implementation "X" interoperable with implementation "Y" of "Z" protocol?
 
+## Quickstart
+A basic use of netbench is demonstrated in the `run-netbench-request-response.sh` script. This script will
```

Comment:
Hmm, not neatly? The toolchain overrides are different in the main quic folder vs the netbench folder. So anything that builds netbench has to be invoked from the netbench directory because it needs MSRV `1.62`. 

We could do `../scripts/netbench-request-response.sh`, but it would be the only script in the `script` directory that can't be invoked from the main project repo, which seems confusing.

We could override the toolchain with `cargo +stable build --release etc` to get around the relative path thing. It might be a bit confusing for people because that will trigger a recompilation of everything, but I prefer that to the weird path invocation.

Reply:
sorry i'm suggesting we put it in `{root}/netbench/scripts`. ideally, netbench becomes its own repo eventually so it would be good to have everything self-contained :)